OvieMudi/Questioner__
# #163499137 migrate project to postgres

#### What does this PR do?
- Migrate all API routes to Postgres databases
- Initializes project for data persistence and authentication
#### Description of Task to be completed?
Make sure the following endpoint are working
```
POST /api/v1/users
GET /api/v1/users
GET /api/v1/users/:id
PATCH /api/v1/users/:id
DELETE /api/v1/users/:id

POST /api/v1/meetups
GET /api/v1/meetups
GET /api/v1/meetups/?scope=upcoming
GET /api/v1/meetups/:id
PATCH /api/v1/meetps/:id
POST /api/v1/meetups/:id/rsvps
GET /api/v1/meetups/:id/rsvps
DELETE /api/v1/meetups/:id

POST /api/v1/questions
GET /api/v1/questions
GET /api/v1/questions/:id
PATCH /api/v1/questions/:id
PATCH /api/v1/questions/:id/upvote
PATCH /api/v1/questions/:id/downvote
DELETE /api/v1/questions/:id
```
#### How should this be manually tested?
- From the command line run `npm run migrate`
- Start the server and make sure all the endpoints above are working
#### What are the relevant pivotal tracker stories?
[#163499137](https://www.pivotaltracker.com/story/show/163499137)
